releasenotes/notes/proxy-37c94e2fb428c6fc.yaml

```diff
@@ -0,0 +1,7 @@
+---
+features:
+  - |
+    /v0.4/trace requests can now be proxied to an actual agent by passing an
+    agent url to the test agent. This can be done by passing the `--agent-url`
+    command-line option or via the `DD_AGENT_URL` or `DD_TRACE_AGENT_URL`
+    environment variables.
```

Comment:
```suggestion
    /v0.4/trace requests can now be proxied to an actual agent by passing an
    agent url to the test agent. This can be done by passing the ``--agent-url``
    command-line option or via the ``DD_AGENT_URL`` or ``DD_TRACE_AGENT_URL``
    environment variables.
```